app/scripts/modules/core/src/pipeline/config/stages/googleCloudBuild/GoogleCloudBuildStageForm.tsx

```diff
@@ -144,6 +195,34 @@ export class GoogleCloudBuildStageForm extends React.Component<
             updatePipeline={this.props.updatePipeline}
           />
         )}
+        {stage.buildDefinitionSource === buildDefinitionSources.TRIGGER && (
+          <React.Fragment>
```

Comment:
Feel free to ignore this, but just a heads-up that you can just use the fragment shorthand here -- only time you should need explicit `React.Fragment` is if you need to key the element when it's spit out as a result of mapping some collection.
```suggestion
          <>
```

Reply:
Cool... has been a while since I wrote react code. Thanks for the suggestion.